commissions/views.py

```diff
@@ -0,0 +1,16 @@
+from django.shortcuts import render, get_object_or_404
+from .models import Commission, Comment
+
+def commission_list(request):
+    commissions = Commission.objects.all().order_by('-created_on')

```

Comment:
The majority are sorting with a `Meta` subclass, so just use the `Meta` subclass instead.